turtlebot3c-bringup-snap/snap/snapcraft.yaml

```diff
@@ -1,5 +1,5 @@
 name: turtlebot3c-bringup
-adopt-info: turtlebot3c-bringup
+version: '0.1'
```

Comment:
Is there no tag on turtlebot3_bringup to which we can pin this?

Reply:
in our case this is `turtlebot3c_bringup` source code that we are pulling. Not directly `turtlebot3_bringup`. Additionally, turtlebot3-bringup never got tagged on ROS 2 distros

Reply:
You're right.